src/components/content/rich-text/rich-text-query.ts

```diff
@@ -22,7 +22,7 @@ export const ContentfulRichText = graphql`
           contentType
           url
         }
-        gatsbyImageData(width: 1440)
+        gatsbyImageData(layout: CONSTRAINED)
```

Comment:
What did you add this, what is it doing exactly?

Reply:
This option instructs the Gatsby Image plugin not to scale images beyond their resolution, so they won't appear overly blurry. It still respects the enclosing container though, thus images are never rendered wider than the rich-text content.

Reply:
ah okay, thank you 👍